backend/ipsec/handle_xfrm.go

```diff
@@ -47,14 +47,20 @@ func AddXFRMPolicy(myLease, remoteLease *subnet.Lease, dir netlink.Dir, reqID in
 		Reqid: reqID,
 	}
 
-	log.Infof("Adding ipsec policy: %+v", tmpl)
-
 	policy.Tmpls = append(policy.Tmpls, tmpl)
 
-	if err := netlink.XfrmPolicyAdd(&policy); err != nil {
-		return fmt.Errorf("error adding policy: %+v err: %v", policy, err)
+	existingPolicy, err := netlink.XfrmPolicyGet(policy)
+	if err != nil {
```

Comment:
Is there a way we can confirm that 'err' is a NotFound error?

Reply:
Good point, fixed!